users/models.py

```diff
@@ -86,8 +86,8 @@ class Membership(models.Model):
     postal_code = models.CharField(max_length=20, blank=True)
 
     # PSF fields
-    psf_code_of_conduct = models.NullBooleanField('I agree to the PSF Code of Conduct', blank=True)
-    psf_announcements = models.NullBooleanField('I would like to receive occasional PSF email announcements', blank=True)
+    psf_code_of_conduct = models.BooleanField('I agree to the PSF Code of Conduct', blank=True, null=True)
+    psf_announcements = models.BooleanField('I would like to receive occasional PSF email announcements', blank=True, null=True)
```

Comment:
Since this was deprecated only since 3.1, I think it's better to keep the `NullBooleanField` instead. That way, we'll reduce the volume of database migrations within this PR and leave more complex django 3 changes, such as these, to a subsequent PR.

Reply:
I consider this change simple because it doesn't generate SQL.

```python
$ python manage.py sqlmigrate users 0014
BEGIN;
--
-- Alter field psf_announcements on membership
--
--
-- Alter field psf_code_of_conduct on membership
--
--
-- Alter field first_name on user
--
ALTER TABLE "users_user" ALTER COLUMN "first_name" TYPE varchar(150);
COMMIT;

```

Should I keep the NullBooleanField and create a new pull-request?

Changing the `first_name` is only necessary for when upgrading to Django 3.1, I will remove it

Reply:
Django 2.1 release notes also recommend this change https://docs.djangoproject.com/en/3.2/releases/2.1/#models

Reply:
Got it! So I guess we can keep this since there's no SQL being ran. Thanks for removing the `first_name` migration too.